rotkehlchen/chain/ethereum/modules/thegraph/decoder.py

```diff
@@ -25,3 +51,90 @@ def __init__(
             native_asset=A_GRT,
             staking_contract=CONTRACT_STAKING,
         )
+
+    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a transfer delegating GRT from ethereum to arbitrum"""
+        if context.tx_log.topics[0] != DELEGATION_TRANSFERRED_TO_L2:
```

Comment:
is there something new in this code? Or is it all moved from the other file to here?

Reply:
Just moved